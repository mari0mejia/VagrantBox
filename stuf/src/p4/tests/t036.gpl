// test every single parameter except animation_block
string window_title = "Project 4 t036.gpl ***PRESS q TO QUIT***";

rectangle r(x = 100,
            y = 100,
            w = 100,
            h = 100,
            visible = 1,
            red = 0.2,
            green = 0.3,
            blue = 0.4,
            proximity = 10,
            drawing_order = 20,
            rotation = 45.2,
            user_int = 1,
            user_int2 = 2,
            user_int3 = 3,
            user_int4 = 4,
            user_int5 = 5,
            user_double = 1.1,
            user_double2 = 2.2,
            user_double3 = 3.3,
            user_double4 = 4.4,
            user_double5 = 5.5,
            user_string = "one",
            user_string2 = "two",
            user_string3 = "three",
            user_string4 = "four",
            user_string5 = "five"
           );

circle c(x = 250,
         y = 300,
         radius = 50,
         visible = 1,
         red = 0.4,
         green = 0.2,
         blue = 0.8,
         proximity = 10,
         drawing_order = 20,
         user_int = 1,
         user_int2 = 2,
         user_int3 = 3,
         user_int4 = 4,
         user_int5 = 5,
         user_double = 1.1,
         user_double2 = 2.2,
         user_double3 = 3.3,
         user_double4 = 4.4,
         user_double5 = 5.5,
         user_string = "one",
         user_string2 = "two",
         user_string3 = "three",
         user_string4 = "four",
         user_string5 = "five"
        );
 
triangle t(x = 100,
           y = 300,
           size = 75,
           skew = .75,
           visible = 1,
           red = 0.4,
           green = 0.2,
           blue = 0.4,
           proximity = 10,
           drawing_order = 20,
           rotation = 22.2,
           user_int = 1,
           user_int2 = 2,
           user_int3 = 3,
           user_int4 = 4,
           user_int5 = 5,
           user_double = 1.1,
           user_double2 = 2.2,
           user_double3 = 3.3,
           user_double4 = 4.4,
           user_double5 = 5.5,
           user_string = "one",
           user_string2 = "two",
           user_string3 = "three",
           user_string4 = "four",
           user_string5 = "five"
          );


textbox x(x = 250,
          y = 100,
          text = "Yikes!",
          size = .5,
          visible = 1,
          red = 0.8,
          green = 0.4,
          blue = 0.3,
          proximity = 10,
          drawing_order = 20,
          user_int = 1,
          user_int2 = 2,
          user_int3 = 3,
          user_int4 = 4,
          user_int5 = 5,
          user_double = 1.1,
          user_double2 = 2.2,
          user_double3 = 3.3,
          user_double4 = 4.4,
          user_double5 = 5.5,
          user_string = "one",
          user_string2 = "two",
          user_string3 = "three",
          user_string4 = "four",
          user_string5 = "five"
         );
