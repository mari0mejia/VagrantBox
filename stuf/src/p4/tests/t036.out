gpl.cpp::main()
  input file(tests/t036.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
circle c
{
    animation_block = NULL,
    blue = double(0.8),
    drawing_order = int(20),
    green = double(0.2),
    h = int(100),
    proximity = int(10),
    radius = int(50),
    red = double(0.4),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(100),
    x = int(250),
    y = int(300)
}
rectangle r
{
    animation_block = NULL,
    blue = double(0.4),
    drawing_order = int(20),
    green = double(0.3),
    h = int(100),
    proximity = int(10),
    red = double(0.2),
    rotation = double(45.2),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(100),
    x = int(100),
    y = int(100)
}
triangle t
{
    animation_block = NULL,
    blue = double(0.4),
    drawing_order = int(20),
    green = double(0.2),
    h = int(62),
    proximity = int(10),
    red = double(0.4),
    rotation = double(22.2),
    size = int(75),
    skew = double(0.75),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(75),
    x = int(100),
    y = int(300)
}
string window_title = "Project 4 t036.gpl ***PRESS q TO QUIT***"
textbox x
{
    animation_block = NULL,
    blue = double(0.3),
    drawing_order = int(20),
    green = double(0.4),
    h = int(11),
    proximity = int(10),
    red = double(0.8),
    size = double(0.5),
    spacing = int(10),
    text = string("Yikes!"),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(10),
    x = int(250),
    y = int(100)
}
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program. Printing the symbol table.
circle c
{
    animation_block = NULL,
    blue = double(0.8),
    drawing_order = int(20),
    green = double(0.2),
    h = int(100),
    proximity = int(10),
    radius = int(50),
    red = double(0.4),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(100),
    x = int(250),
    y = int(300)
}
rectangle r
{
    animation_block = NULL,
    blue = double(0.4),
    drawing_order = int(20),
    green = double(0.3),
    h = int(100),
    proximity = int(10),
    red = double(0.2),
    rotation = double(45.2),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(100),
    x = int(100),
    y = int(100)
}
triangle t
{
    animation_block = NULL,
    blue = double(0.4),
    drawing_order = int(20),
    green = double(0.2),
    h = int(62),
    proximity = int(10),
    red = double(0.4),
    rotation = double(22.2),
    size = int(75),
    skew = double(0.75),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(75),
    x = int(100),
    y = int(300)
}
string window_title = "Project 4 t036.gpl ***PRESS q TO QUIT***"
textbox x
{
    animation_block = NULL,
    blue = double(0.3),
    drawing_order = int(20),
    green = double(0.4),
    h = int(51),
    proximity = int(10),
    red = double(0.8),
    size = double(0.5),
    spacing = int(10),
    text = string("Yikes!"),
    user_double = double(1.1),
    user_double2 = double(2.2),
    user_double3 = double(3.3),
    user_double4 = double(4.4),
    user_double5 = double(5.5),
    user_int = int(1),
    user_int2 = int(2),
    user_int3 = int(3),
    user_int4 = int(4),
    user_int5 = int(5),
    user_string = string("one"),
    user_string2 = string("two"),
    user_string3 = string("three"),
    user_string4 = string("four"),
    user_string5 = string("five"),
    visible = int(1),
    w = int(195),
    x = int(250),
    y = int(100)
}
